site/content/zh-CN/docs/tasks/manage/productization/prometheus.md

```diff
@@ -1,64 +1,63 @@
 ---
-title: "Configure Prometheus"
+title: "配置 Prometheus"
 date: 2025-03-14
 weight: 2
 description: >
-  Prometheus Support
+  Prometheus 支持
 ---
 
+此页面展示了如何配置 Kueue 以使用 Prometheus 指标。
 
-This page shows how you configure Kueue to use prometheus metrics.
+此页面适用于[批处理管理员](/zh-CN/docs/tasks#batch-administrator)。
```

Comment:
For simplicity and consistency, we can replace `zh-CN` with `zh` globally later.